docs/MockFunctionAPI.md

```diff
@@ -102,11 +102,12 @@ is available to clear mocks automatically between tests.
 
 ### `mockFn.mockReset()`
 
-Resets all information stored in the mock, including any initial implementation
-and mock name given.
+Does everything that [`mockFn.mockClear()`](#mockfnmockclear) does, and also
+removes any mocked return values.
 
-This is useful when you want to completely restore a mock back to its initial
-state.
+This is useful when you want to completely restore a _mock_ back to its initial
+state. (Note that resetting a _spy_ will result in a function with no return
+value).
```

Comment:
I wouldn't use the word "restore" here, what do you think about using "completely reset a mock"

Reply:
Yes, I agree :) Will change